config/standard/package.json

```diff
@@ -14,13 +14,13 @@
   },
   "devDependencies": {
     "@medic/contact-summary-testing": "^2.0.1",
-    "chai": "^4.1.2",
-    "chai-exclude": "^1.0.8",
-    "eslint": "^6.5.1",
-    "eslint-plugin-json": "^1.3.2",
-    "cht-conf": "^3.7.0",
-    "cht-conf-test-harness": "^2.4.1",
-    "medic-conf-test-harness": "^0.1.19",
-    "mocha": "^5.1.1"
+    "chai": "^4.3.8",
+    "chai-exclude": "^2.1.0",
+    "eslint": "^8.49.0",
+    "eslint-plugin-json": "^3.1.0",
+    "cht-conf": "3.18.3",
```

Comment:
I know we're getting rid of the standard config but while we're here updating dependencies, is there anything preventing us from upgrading `cht-conf` to the latest version here?

Reply:
Yeah, honestly now I cannot remember what it was, but something did break when going past this version...  As you said, though, the standard config is going with the :dodo: so no need to worry about this!